pkg/config/sample-config.txt

```diff
@@ -0,0 +1,26 @@
+version = 31
+credential.accessKey = xxx
+credential.secretKey = yyy    ## SecretKey cannot be changed
+region = us-east-1
+browser = on
+worm = on
+cache.drives = 
+cache.expiry = 111
```

Comment:
Expiry should be 90 default

Reply:
OK

Reply:
No we don't need to.. but whatever is mentioned should have default values.